src/bootstrap/mod.rs

```diff
@@ -0,0 +1,88 @@
+use bindings::Windows::Win32::{
+    Foundation::HWND,
+    UI::WindowsAndMessaging::{MessageBoxW, MB_ICONERROR, MB_OK},
+};
+
+mod tests;
```

Comment:
Unit tests are normally in the same file (in a submodule). This is fine, but perhaps not the most idiomatic.